utils/build/docker/ruby/rack/config.ru

```diff
@@ -257,16 +258,28 @@ class Users
   end
 end
 
+# /rasp/ssrf
+class SSRFHandler
```

Comment:
Could it be a module?

Reply:
yes, but then I need to change all those endoints to modules